vms/evm/destination_client.go

```diff
@@ -9,27 +9,29 @@ import (
 	"context"
 	"math/big"
 	"sync"
+	"time"
 
 	"github.com/ava-labs/avalanchego/ids"
 	"github.com/ava-labs/avalanchego/utils/logging"
 	avalancheWarp "github.com/ava-labs/avalanchego/vms/platformvm/warp"
-	"github.com/ava-labs/coreth/rpc"
```

Comment:
Unrelated to this PR but swapped `subnet-evm/rpc` in for `coreth/rpc` since my IDE was complaining that coreth should be direct dependency and this was the only place it was required explicitly.

Nor sure why the `go mod tidy` check didn't pick it up in CI